documentation/movies.workspace.adoc

```diff
@@ -0,0 +1,731 @@
+= Movies graph
+:imagesdir: https://neo4j-graph-examples.github.io/get-started/documentation/img/
+
+== Movies graph
+
+This guide uses pop-cultural connections to teach you how to use Cypher to create and query data.
+You will explore connections between actors, directors, and movies, and also solve the famous Bacon-path.
+
+Note that this guide assumes that you use an empty database as you will modify it along the way.
+You will learn how to delete the data you create towards the end of the guide as well.
+
+=== What you will learn
+
+This guide takes around 10 minutes and by the end of it you will be able to:
+
+- Create nodes and relationships that connect them.
+- Find individual nodes by using conditional matches.
+- Find patterns in the graph.
+- Understand and use the shortest path algorithm.
+- Write a basic recommendation query.
+- Delete all data.
+
+Let's get started!
+
+In the next step you are going to create the data you'll be working with.
+
+== Create the data
+
+The Cypher `CREATE` clause is used to create data in your graph.
+If you are not familiar with Cypher syntax, you can try the _Query fundamentals_ guide or see the link:https://neo4j.com/docs/cypher-manual/current[Cypher Manual].
+
+You are going to create a dataset with `Person` and `Movie` nodes and different types of relationships to connect them.
+
+image::movie-model.png[]
+
+Use the command below but be careful to only run it **once** as it will add data every time it is run.
+
+[source,cypher]
+----
+CREATE (TheMatrix:Movie {title:'The Matrix', released:1999, tagline:'Welcome to the Real World'})
```

Comment:
Should we switch the CREATE statements to MERGE in the setup, so running it multiple times will not mess it up?

Reply:
It would be convenient but as we are talking about creating data, it makes more sense to use `CREATE`, IMHO.